transform_dialect/examples/accel/matmul_codegen_spec_pad.mlir

```diff
@@ -52,6 +52,7 @@ module attributes { transform.with_named_sequence } {
       pack_paddings=[1, 1, 0],
       copy_back_op="none"
     } : (!transform.any_op) -> (!transform.any_op, !transform.any_op, !transform.any_op)
+    %pad_dps = transform.structured.rewrite_in_destination_passing_style %pad : (!transform.any_op) -> !transform.any_op
```

Comment:
In `transform.structured_tile_to_forall_op` dont you need to pass `pad_dps` instead of `padded` ?

Reply:
I don't think so. The pad operation (%pad) is rewritten in destination style, and we need to pass the padded operation (%padded) to the next transform.

Reply:
Ah, ok... I see your point. Thanks...